app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>Hello! this is the user show</h1>
\ No newline at end of file
+<div class="container">
+  <div class="user_info">
+    <div class="img">
+      <%= image_tag @user.photo, alt: "User Photo", class: "user-photo"%>
+    </div>
+    <div class="user">
```

Comment:
- [ ] Fantastic progress so far! 💪✨ Please ensure the image styling aligns precisely with the wireframe guidelines Currently, when I'm using a valid image URL, this is how the image is being displayed 👇

![image](https://github.com/evansnyamekye/blog-app/assets/106119105/db298997-11aa-47d2-b8d0-8dba0a8be257)